textfinding_indikator1.py

```diff
@@ -0,0 +1,119 @@
+# this code is for TEXT FINDING - INDIKATOR 1
+# input is txt file from preprocess dokumen lama, nama instansi & judul from dokumen lama & baru
+
+import re
+import preprocess_dokbaru as dokbaru
+import preprocess_doklama as doklama
+from utility import *
+
+
+def txtreader(filename, lv, keyword):
+    # func to search keyword in txt file
+
+    # open txt file
+    file = open(f'cleaned_{filename}.txt', 'r')
+
+    idx = 0
+    result = []
+
+    # read line by line from txt
+    for line in file:
+
+        if (lv == 1):
+            if re.search(keyword[0], line, re.IGNORECASE):
+                result.append([idx, line])
+
+        elif (lv == 2):
+            # cek dgn keyword 'domain'. cont: domain arsitektur proses bisnis, domain data dan informasi, domain arsitektur keamanan, dll
+            for key in list(keyword):
+                if key in keyword:
+
+                    # manfaatin regex untuk searching yg not only kata keyword
+                    reg = f'(?:(domain)\s(arsitektur)?\s?({key}))'
```

Comment:
Maybe we can modify every regex literal a lil bit to include `*` (asterisk) symbol so that it can handle cases with any number of whitespaces in between the key words. Just like here https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/7#discussion_r1141603066